### PR TITLE
Signal shape change

### PR DESCRIPTION
**Pull request by maikia**
Changes signal from the sinusoidal to squared step with random amplitudes of activated signal in each parcel

### CODE REVIEW

File: simulation/raw_signal.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
-import numpy as np
 import mne
+import numpy as np
+
+np.random.seed = 42
```
Comment (agramfort) [inline simulation/raw_signal.py]: please use random_state and not a global state

File: simulation/raw_signal.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
-import numpy as np
 import mne
+import numpy as np
```
Comment (agramfort) [inline simulation/raw_signal.py]: I tend to order import by order of dependencies.

so np is first

File: simulation/raw_signal.py [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+
+np.random.seed = 42
 
 
 def generate_signal(raw_fname, fwd_fname, subject, parcels, n_events=30,
                     signal_type='eeg'):
-    signal_len = 10
+    signal_len = 0.1  # in sec
```
Comment (agramfort) [inline simulation/raw_signal.py]: is it necessary to simulate for such a long time? a few ms should be enough

**Review by agramfort — APPROVED** [review]
LGTM

if results of starting kit makes sense +1 for MRG

**Review by hichamjanati — APPROVED** [review]

**Comment by maikia** [comment]
@agramfort starting kit passes through alright. 
Thanks!
Merging

**Pull request merged**